.circleci/config.yml

```diff
@@ -0,0 +1,127 @@
+# Java Gradle CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-java/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+    # specify the version you desire here
+    - image: rocker/verse:4.1.1
```

Comment:
Suggest to bump R version to [4.4.0](https://hub.docker.com/layers/rocker/verse/4.4.0/images/sha256-9acf80b03b866b1faef241b50ac7025ea42263b673954102c11eb09a24fb96d3?context=explore).